source/partials/release_notes/_release-17-2-0.md.erb

```diff
@@ -0,0 +1,119 @@
+#### Authorization Plugins (beta)
+
+Currently GoCD provides multiple ways of Authentication like File based Authentication, LDAP/AD Authentication and Plugin Based Authentication. Apart from Authentication, GoCD provides ability to restrict certain operations to specific Users and Roles and manage them. Currently, roles can be managed only through GoCD and it does not provide an ability to use ACL’s defined in systems used for authentication. With this limitation, administrators need to configure ACL's in multiples places.
+
+With the introduction of Authorisation Plugin(Beta)  GoCD is moving in a direction where Authentication/Authorisation is moved out of GoCD into plugins. With this, the Authorisation Plugins would be responsible to provide Identity Service. The plugins have the flexibility to use any identity service providers like LDAP, Github etc.
+
+
+[LDAP Authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) is in place built on this Authorization endpoint. Please checkout this feature and provide us feedback to enhance this endpoint.
+
+#### Feature Enhancements
+
+* <%= link_to_commit 'c08843512b494ef1c483ebb699c615170dd13cf0', "Template Authorization UI and API enhancements" %>
+
+* No restriction on template view config. All pipeline group admins by default provided view permission to templates. Super admin can restrict this access.
+* Super admin can provide template view access to users/roles.
+* Allow template admins to delete templates via UI and API
+
+<figure>
+  <%= image_tag 'releases/17.2.0/template-permissions.png', alt: 'Template Permissions' %>
+</figure>
+
+#### Bug Fixes
+
+* <%= link_to_commit '7ddf84eaaf813546aa1f8eb684ef9cf3187f6617', 'User search functionality - Search through plugins implementing AuthenticationExtension.', 3107 %>
+* <%= link_to_commit 'a837fc05cca60a9bbb8dcd7d3a3c7b1228a51358', 'Sort resources and environments alphabetically on agents page selector dropdown.', 3134 %>
+* Multiple UI fixes on the old and new themes.
```

Comment:
There are multiple UI fixes, couldn't find any PR which is major in those. That's why didn't add any PR.